raidenchannels/raidenmanager.go

```diff
@@ -0,0 +1,165 @@
+package raidenchannels
+
+import (
+	"math/big"
+	"sync"
+	"time"
+
+	"github.com/ExchangeUnion/xud-tests/ethclient"
+
+	"github.com/ExchangeUnion/xud-tests/xudclient"
+
+	"github.com/ExchangeUnion/xud-tests/raidenclient"
+	"github.com/ExchangeUnion/xud-tests/slackclient"
+)
+
+// TODO: clean inactive channels
+
+type token struct {
+	address       string
+	channelAmount uint64
+}
+
+var channelTokens = []token{
+	{
+		address:       "123",
+		channelAmount: 123,
+	},
+	{
+		address:       "987",
+		channelAmount: 987,
+	},
+}
+
+var raidenChannels = make(map[string]bool)
```

Comment:
Maybe call this `hasRaidenChannels` or `peerHasChannelsMap`? Currently the name makes it sound like a list of raiden channels.